### ISSUE TITLE
Auto-Complete filter

### ISSUE THREAD

**Issue by ckarmy** [yairEO/tagify#181]
Hi! if I start typing:
![screen shot 2019-01-15 at 9 26 42 pm](https://user-images.githubusercontent.com/10090783/51218720-6b2e2900-190c-11e9-9437-f68118da9167.png)

Work perfect! But if I type:
![screen shot 2019-01-15 at 9 26 56 pm](https://user-images.githubusercontent.com/10090783/51218746-7d0fcc00-190c-11e9-99dc-031be43a7a10.png)

I have no results in my auto complete but I need that in this case value "cristian karmy" appear if I type "Cr" or "Ka". But only I get results with "Cr" that is the start of the value.

Its posible to change the "sensibility" of the auto complete but in others words of the value?

Thanks!

**Comment by tvucak** [comment]
Your Ticket is a duplicate of https://github.com/yairEO/tagify/issues/164

**Comment by yairEO** [comment]
what do you expect to see, in the gray text (auto-completion), which comes right after `ka` ?

**Comment by ckarmy** [comment]
> what do you expect to see, in the gray text (auto-completion), which comes right after `ka` ?

I expect that if I type `ka` the auto-completion suggest `cristian karmy`

**Comment by yairEO** [comment]
That is not auto complete, that is simply a suggestion mechanism, which is different than auto-complete

**Comment by ckarmy** [comment]
> That is not auto complete, that is simply a suggestion mechanism, which is different than auto-complete

You are right! But its possible to suggest like that in tagify?

**Comment by yairEO** [comment]
yes it is possible, because the dropdown can show such suggestion, but regarding the auto-complete, it would look weird if typing `ka` would autocomplete to:

**ka**cristian karmy

**Comment by ckarmy** [comment]
@yairEO I dont know how to enable fuzzy search on `tagify.min.js`
